app/models/chef.rb

```diff
@@ -1,4 +1,18 @@
 class Chef < ApplicationRecord
-   validates_presence_of :name
-   has_many :dishes
+  validates_presence_of :name
+  has_many :dishes
+
+  def unique_ingredients_used
```

Comment:
Nice work setting up this query! Keep in mind that you have an association set up between dishes and ingredients already, so you could consolidate the joins down to `Ingredient.joins(:dishes)`.

You could also choose to leverage an additional association from this Chef model. Chefs have many dishes, so you can also take advantage of their associations. In this  model, you could add `has_many :ingredients, through: :dishes`. Then from there, you could just do `ingredients.distinct` from a chef instance to grab all the unique ingredients a given chef uses